fusestore/store/src/meta_service/state_machine.rs

```diff
@@ -75,6 +79,9 @@ pub struct StateMachine {
     /// table id to table mapping
     pub tables: BTreeMap<u64, Table>,
 
+    ///
+    pub tbl_parts: HashMap<String, HashMap<String, Vec<DataPartInfo>>>,
```

Comment:
@drmingdrmer Should I remove this field and replace table<->parts management to parts field in `Table`.

Reply:
Yeah I'd like to see a non-nested mapping :DDD

If you'd like to move things forward quickly, it's all right to open another PR fixing this issue.

Reply:
Created a related issue: https://github.com/datafuselabs/datafuse/issues/1102